utils/cargo-tfhe-lints/src/main.rs

```diff
@@ -0,0 +1,20 @@
+use std::process::{exit, Command};
+
+fn main() {
+    let cargo_args = std::env::args().skip(2).collect::<Vec<_>>();
+    let toolchain = "+nightly-2024-05-02";
```

Comment:
can the toolchain choice be automated with an include! and some compile time processing ?

Reply:
done